frontend/components/EventDetails.js

```diff
@@ -19,6 +19,14 @@ class EventDetails extends React.Component {
               (this.props.event.link).includes("http://") || (this.props.event.link).includes("https://") ?
                 this.props.event.link : "http://" + this.props.event.link}> {this.props.event.link}
             </a></p> }
+            {this.props.event.additional_background_check  && <p> <b>Additional Background Check Info: </b>
+            {this.props.event.additional_background_check}</p> }
+
+            {this.props.user.bio.role == "volunteer" ?
+                (this.props.event.volunteers.includes(this.props.user._id) ?
+                    <button type="button" onClick={()=>this.props.unSignUp()}> Un-Sign Up </button> :
```

Comment:
I think the button should say something more like "Cancel Sign-up" rather than "Un-Sign Up," unsign up is fine for code purposes though